packages/ui/src/composables/useDropdown.ts

```diff
@@ -169,6 +151,25 @@ export const useDropdown = (
   }
 
   watchPostEffect(() => {
+    const rootRef = computed(() => {
```

Comment:
Computed must be created in setup function, not in sub function. This cause creating new computed each time anchor or content DOM rect change.

Reply:
What if replace it with a ref?